### ISSUE TITLE
coursory.com needs to be updated for the 2015-2016 academic year

### ISSUE THREAD

**Issue by zmwangx** [karthikv/coursory#2]
15-16 autumn data is out today.

**Comment by karthikv** [comment]
Hmm, I'd be happy to update, but it seems like the ExploreCourses API isn't returning valid data for course schedules anymore. For instance, take a look at http://explorecourses.stanford.edu/search?view=xml&academicYear=&page=0&q=ACCT&filter-departmentcode-ACCT=on&filter-coursestatus-Active=on&filter-term-Autumn=on. Notice how the `<schedule>` tag has 'crsearch.action.SectionSchedule@659ba7ac' in it as opposed to the actual dates/times of the course.

I'm not sure why/when this breaking change was made. I could simply not display schedules on coursory, but they're an important component.

**Comment by zmwangx** [comment]
Yeah I also noticed the problem when running a local instance. Not sure why.
